src/main/resources/camel/odoo-add-extra-customer-details.xml

```diff
@@ -10,7 +10,7 @@
                 <simple>${properties:odoo.dob.field:null} != null &amp;&amp; ${properties:odoo.dob.field:null} != ''</simple>
 
                 <setProperty name="odooDobField">
-                    <simple>{{odoo.dob.field}}</simple>
+                    <simple>{{properties:odoo.dob.field}}</simple>
```

Comment:
Is this meant to address a bug?

Reply:
Yes.

Reply:
I think this is actually introducing a bug... @icrc-loliveira have you tested this? Or did you intend to do as below?
```suggestion
                    <simple>${properties:odoo.dob.field}</simple>
```
Which kind of bug is being fixed here though?